### PR TITLE
Tip tap RTE to PDF download

### PR DESCRIPTION
**Pull request by mdalpozzo**

### CODE REVIEW

File: src/components/RichTextEditor/TipTapEditor.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+                    'w-[794px]', // Width of A4 paper in pixels
+                    'm-auto p-[20px]', // Centered and padding inside the editor to simulate margins
+                    // Allows scrolling within the editor if the content exceeds A4 height
+                    // TODO: display a line in the editor representing the end of the page
+                    'overflow-y-auto',
+                    'shadow-md'
+                ),
+            },
+        },
+        content: resumeHTML,
```
Comment (FullstackJack) [inline src/components/RichTextEditor/TipTapEditor.tsx:39]: Could we just pass this dummy HTML in via props from the testEditor?

Comment (FullstackJack) [reply src/components/RichTextEditor/TipTapEditor.tsx:39]: Let's just add `content` to the `TipTapEditorProps`.

**Review by FullstackJack — APPROVED** [review]
Let's just update that one prop on the richtexteditor and good to merge.

File: src/components/RichTextEditor/TipTapEditor.tsx (hunk 1 above)
Comment (mdalpozzo) [reply src/components/RichTextEditor/TipTapEditor.tsx:39]: Yes, totally.  Dependency injection ftw!  I’ll do this in a few.  The editor still needs a lot of work and design feel free to muck with it

**Pull request merged**